src/api/tests/common.py

```diff
@@ -31,7 +32,7 @@
 from django.core.files.uploadedfile import InMemoryUploadedFile
 
 RESET = "reset"
-
+ME_DEFAULT_TEST_IMAGE = "https://www.whitehouse.gov/wp-content/uploads/2021/04/P20210303AS-1901-cropped.jpg"
```

Comment:
you know this image is a joke, because Sam thinks I look like Joe Biden

Reply:
🤣 🤣 🤣 🤣 okay great. I thought Biden was the core team's favorite guy 🤣 🤣. Then we will change it to something more light weight so that tests run faster.